tests/queries/0_stateless/01730_distributed_group_by_no_merge_order_by_long.sql

```diff
@@ -4,15 +4,15 @@ drop table if exists data_01730;
 
 -- does not use 127.1 due to prefer_localhost_replica
 
-select * from remote('127.{2..11}', view(select * from numbers(1e6))) group by number order by number limit 20 settings distributed_group_by_no_merge=0, max_memory_usage='100Mi'; -- { serverError 241 }
+select * from remote('127.{2..11}', view(select * from numbers(1e6))) group by number order by number limit 20 settings distributed_group_by_no_merge=0, max_memory_usage='200Mi'; -- { serverError 241 }
```

Comment:
This is wrong. It was me testing to see where the memory raised to adapt the second query (which is still tricky to adjust).